cache/in-memory/src/shard.rs

```diff
@@ -431,6 +433,7 @@ impl UpdateCache<InMemoryCache, InMemoryCacheError> for Event {
             ReactionAdd(v) => c.update(v).await,
             ReactionRemove(v) => c.update(v).await,
             ReactionRemoveAll(v) => c.update(v).await,
+            ReactionRemoveEmoji(_v) => Ok(()),
```

Comment:
I guess this is one of the todos?

Reply:
This is going to be done in a larger PR for #25 and #65, which will implement all of the missing cache implementations.